src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -67,6 +67,37 @@ const convertEntityIdsToTags = (ids: string[], entities: CLM.EntityBase[]): OF.I
         .map<OF.ITag>(convertEntityToTag)
 }
 
+const toConditionName = (entity: CLM.EntityBase, enumValue: CLM.EnumValue): string => {
+    return `${entity.entityName} = ${enumValue.enumValue}`
```

Comment:
Seems really weird to have the save variable twice.  I'm guessing the parent isn't actually an enumValue it's the container of the value?

I thought it was like:
```
{
  id: "adfadf"
  value: "asdfasdf"
}
```

Reply:
Yeah, it's an EnumValue object.   Shahin asked to the name the value "enumValue" too, so it's enumValue.enumValue